prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      min-height: 100vh;
+      align-items: center;
+      justify-content: center;
+    }
+    .star-wars-card {
+      flex-basis: calc(100% / 3);
+    }
```

Comment:
Hi Joel, since the `.card-wrapper` is the child of `#card-deck`, you'll want to apply your `flex-basis` to that, instead of the `.star-wars-card`.

Reply:
<img width="1440" alt="Screen Shot 2022-02-26 at 9 12 24 AM" src="https://user-images.githubusercontent.com/97650794/155852409-273908bf-9df8-4439-b6f7-0d84c1f6842e.png">
<img width="1440" alt="Screen Shot 2022-02-26 at 9 12 31 AM" src="https://user-images.githubusercontent.com/97650794/155852416-2e384c3b-f34b-40dc-b19c-7a313ef2d081.png">
<img width="1440" alt="Screen Shot 2022-02-26 at 9 12 40 AM" src="https://user-images.githubusercontent.com/97650794/155852420-56e4bc94-d8d2-4700-b064-31cd33dee9d7.png">
<img width="1440" alt="Screen Shot 2022-02-26 at 9 12 48 AM" src="https://user-images.githubusercontent.com/97650794/155852423-b02082c7-61e0-4adf-815a-9cd2726fd89e.png">